deploy/templates/bicep/ContainerAppSetup.bicep

```diff
@@ -1,4 +1,4 @@
-@minLength(3)
+@minLength(6)
 @maxLength(16)
 @description('Basename that is used to name provisioned resources. Should be alphanumeric, at least 3 characters and less than 16 characters.')
```

Comment:
Same comment as below. I think it'd need to be done for all bicep files

Reply:
Will do.